### PR TITLE
Add commit to github links, improve git link display

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@spadgett review

File: assets/app/views/_pod-template.html [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
                 <span ng-switch-when="Git">
-                  <span ng-if="build.spec.revision">
+                  <span ng-if="build.spec.revision.git.commit">
                     {{build.spec.revision.git.message}}
-                    <span ng-if="build.spec.revision.git.commit">
-                      (<span title="{{build.spec.revision.git.commit}}" class="small">{{build.spec.revision.git.commit.substr(0, 10)}}</span>)
-                    </span>
+                    (<osc-git-link 
+                      uri="build.spec.source.git.uri"
+                      commit="build.spec.revision.git.commit">{{build.spec.revision.git.commit.substr(0, 7)}}</osc-git-link>)
```
Comment (spadgett) [inline assets/app/views/_pod-template.html]: Might use `{{build.spec.revision.git.commit | limitTo: 7}}` here.

File: assets/app/views/_pod-template.html [hunk 2]
```diff
@@ -25,17 +25,17 @@
...
                     <span ng-if="build.spec.revision.git.author">
                       authored by {{build.spec.revision.git.author.name}}
                     </span>
                   </span>
-                  <span ng-if="!build.spec.revision">
-                    <a href="{{build.spec.source.git.uri}}" target="_blank" data-toggle="tooltip" data-placement="right" title="This is a Git URL that can be used with `git clone`">Git</a>
+                  <span ng-if="!build.spec.revision.git.commit">
+                    <osc-git-link uri="build.spec.source.git.uri">{{build.spec.source.git.uri}}</osc-git-link>
```
Comment (spadgett) [inline assets/app/views/_pod-template.html]: I like this much better than using link label `Git`.

**Comment by spadgett** [comment]
LGTM

**Comment by liggitt** [comment]
[merge]

**Pull request merged**